clusters/build-clusters/01_cluster/openshift-monitoring/cluster-monitoring-config_configmap.yaml

```diff
@@ -19,3 +19,7 @@ data:
          resources:
            requests:
              storage: 10Gi
+    thanosQuerier:
```

Comment:
Can we add a comment with the corresponding BZ so this doesn't stay here forever?

Reply:
Done.